src/Componenents/Main.js

```diff
@@ -80,6 +73,33 @@ class Main extends Component {
     }
   };
 
+  getMovies = async () => {
+
+    try {
+      // This URL is the path to my remote server
+      const movieUrl = `https://city-explorer-eddie.herokuapp.com/movies?searchQuery=${this.state.searchQuery}`
+      // This URL is the path to my local server
+      //const movieUrl = `http://localhost:3001/movies?searchQuery=${this.state.searchQuery}`
+
+      const movieData = await axios.get(movieUrl);
+
+      // let movieArray = theMovies.data.results.map(movie => {
+      //   return movie;
+      // });
```

Comment:
Feedback here: Remove unused code before all PR merges to keep in line with best practices 👍